tron/core/jobgraph.py

```diff
@@ -22,20 +22,28 @@ def __init__(self, config_container):
         self._adj_list = defaultdict(list)
         self._rev_adj_list = defaultdict(list)
 
+        dependent_jobs = defaultdict(list)
```

Comment:
I think we don't need to track _both_ dependent jobs and dependent actions, since we can compute/track the jobs in the action names.